marimo/_runtime/runner/hooks_post_execution.py

```diff
@@ -246,7 +246,7 @@ def _store_state_reference(
 @kernel_tracer.start_as_current_span("broadcast_outputs")
 def _broadcast_outputs(
     cell: CellImpl,
-    runner: cell_runner.Runner,
+    _runner: cell_runner.Runner,
```

Comment:
Looks like not used -- should we remove?

Reply:
Signature is needed for the call (just a general "hook")